### PR TITLE
docs: replace inline code with code blocks

### PR DESCRIPTION
**Pull request by YangFong**

### CODE REVIEW

File: translations/README-cn.md [hunk 1]
```diff
@@ -188,31 +188,33 @@
...
-
-`git rebase jwasham/main`
-
-`git push --set-upstream origin progress`
-
-`git push --force`
+克隆项目到本地
+
+```sh
+git clone git@github.com:<your_github_username>/coding-interview-university.git
```
Comment (jwasham) [inline translations/README-cn.md:202]: this needs to clone this project. Remove <your_github_username>

Comment (YangFong) [reply translations/README-cn.md:202]: I see the English version is the same. Should I change it like this

```suggestion
git clone git@github.com:/coding-interview-university.git
```

Comment (jwasham) [reply translations/README-cn.md:202]: You're right, it is the same on the English version. I don't think that works, though.

Comment (YangFong) [reply translations/README-cn.md:202]: So should we change the English version first, without reference, I don't know how to modify, this is just the translation.

Comment (jwasham) [reply translations/README-cn.md:202]: Thank you! I've updated it: https://github.com/jwasham/coding-interview-university/commit/7c6679f7ac0801ef5487480f1a8cc5dfaea84453

The user has to fork to make their own repo, then clone that.

**Comment by jwasham** [comment]
Thank you!

**Pull request merged**